src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -599,65 +616,41 @@ definitions:
     type: string
     description: >-
       The type of descriptor that represents this version of the tool (e.g. CWL,
-      WDL, or NFL).
+      WDL, or NFL). Note that these files can also include associated Docker/container files 
+      and test parameters that further describe a version of a tool
     enum:
```

Comment:
Maybe this shouldn't be a enum? If a new language comes along, then it would require a new version of the API to handle it.

Reply:
This has been discussed previously in the cloud workstream and we decided to go this route on our APIs. There are pros and cons either way